eoscostarica/guide.eoscostarica.io
# revert: this commit revert ca8b836fbf4df7c34b6c172cc4107c869be6d9ab

### Improve homepage styles

### What does this PR do?

- resolves #439 and #534

### Steps to test
1. Go to guides EOS
2. See changes on homepage

#### CheckList
